src/ipa_pc/mod.rs

```diff
@@ -99,9 +93,13 @@ impl<G: AffineCurve, D: Digest> InnerProductArgPC<G, D> {
             .for_each(|(k_l, k_r)| *k_l += &(k_r.mul(round_challenge)));
     }
 
-    /// Computes an opening proof of multiple check polynomials with a corresponding
-    /// commitment GFin opened at point.
-    /// No segmentation here: Bullet Polys are at most as big as the committer key.
+    /// Computes an opening proof of multiple reduction polynomials `h(`xi`,X)` with a 
+    /// corresponding commitment `G_fin` opened at a challenge `point`.
+    /// Used by a separate aggregation proof to optimize the verification of many 
+    /// dlog opening proofs in batch.
+    /// Note: The entire opening assertions, i.e. the xi's and their G_fin's, are 
+    /// already bound the inner state of the Fiat-Shamir rng.
+    // No segmentation here: Reduction polys are at most as big as the committer key.
     pub fn open_reduction_polynomials<'a>(
         ck: &CommitterKey<G>,
         xi_s: impl IntoIterator<Item = &'a SuccinctCheckPolynomial<G::ScalarField>>,
```

Comment:
A more appropriate name for the struct SuccinctCheckPolynomial would be "ReductionPolynomial"